generic_config_updater/generic_updater.py

```diff
@@ -0,0 +1,469 @@
+import json
+import jsonpatch
+import sonic_yang
+import os
+import copy
+from enum import Enum
+from swsssdk import ConfigDBConnector
+from imp import load_source
+load_source('sonic_cfggen', '/usr/local/bin/sonic-cfggen')
+from sonic_cfggen import deep_update, FormatConverter
+
+YANG_DIR = "/usr/local/yang-models"
+CHECKPOINTS_DIR = "/etc/sonic/checkpoints"
+CHECKPOINT_EXT = ".cp.json"
+
+class JsonChange:
+    # TODO: Implement JsonChange
+    pass
+
+class ConfigLock:
+    def acquire_lock(self):
```

Comment:
Please see if this helps:
https://github.com/Azure/sonic-utilities/pull/857/files

Note: We may have a few bug fixes on top of it.

Reply:
Yeah, this was merged then reverted. I hope we have it merged back to azure soon.